engine/runtime-parser/src/main/java/org/enso/compiler/core/IR.java

```diff
@@ -7,25 +7,59 @@
 import org.enso.compiler.core.ir.Expression;
 import org.enso.compiler.core.ir.IdentifiedLocation;
 import org.enso.compiler.core.ir.MetadataStorage;
+import org.enso.compiler.core.ir.ProcessingPass.Metadata;
 import org.enso.compiler.debug.Debug;
 import scala.Option;
 import scala.collection.immutable.List;
 
 /**
- * [[IR]] is a temporary and fairly unsophisticated internal representation format for Enso
+ * {@link IR} is a temporary and fairly unsophisticated internal representation format for Enso
```

Comment:
- What a wonderful welcoming sentence!
- Maybe we want to change it ;-)

```suggestion
 * {@link IR} is an internal representation format for Enso inspired by IR of Scala2 that the current team
 inherited and has to live with.
```

Reply:
> inspired by IR of Scala2 that the current team
> inherited and has to live with.

I think we could be spared personal animosities in the official javadocs. We all know how much you :heart:  Scala.